test/dot_s/export_malloc_free.s

```diff
@@ -36,4 +36,22 @@ realloc:
 	i32.const	$push0=, 0
 	.endfunc
 .Lfunc_end22:
-	.size	realloc, .Lfunc_end22-free
```

Comment:
It's generated for assemblers to calculate the size of objects, but It's ignored by s2wasm.